plugins/modules/exo_rdns_record.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2019, Lorenz Schori <lo@znerol.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: exo_rdns_record
+short_description: Manages reverse DNS records for Exoscale compute instances.
+description:
+    - Set and unset reverse DNS record on Exoscale instance.
+version_added: "2.10"
+author: "Lorenz Schori (@znerol)"
+options:
+  name:
+    description:
+      - Name of the compute instance
+    required: true
+    type: str
+  content:
+    aliases: [ 'value' ]
+    description:
+      - Reverse DSN name for the cs instance. Required if state=present.
+    type: str
+  state:
+    description:
+      - State of the record.
+    default: present
+    choices: [ 'present', 'absent' ]
+    type: str
+extends_documentation_fragment: cloudstack
```

Comment:
this adds a new dependecy to the cloudstack collection, please change it to

`ngine_io.cloudstack.cloudstack`